jib-core/src/main/java/com/google/cloud/tools/jib/cache/CacheStorageWriter.java

```diff
@@ -75,21 +77,36 @@ private WrittenLayer(
    * @throws IOException if an I/O exception occurs
    */
   private static void moveIfDoesNotExist(Path source, Path destination) throws IOException {
-    // If the file already exists, we skip renaming and use the existing file. This happens if a
-    // new layer happens to have the same content as a previously-cached layer.
-    if (Files.exists(destination)) {
+    // Some Windows users report java.nio.file.AccessDeniedException that we suspect is caused
+    // by anti-virus programs, like Windows Defender, that open new files for scanning.
+    // Retry the rename up to 5 times, with 15ms pause between each retry.
+    boolean success =
```

Comment:
Should this run exclusively on windows machines?

Reply:
It shouldn't matter.  There is a possibility this applies to other platforms.  I don't have a strong opinion on this.

Reply:
I can live with that.